packages/block-template/README.md

```diff
@@ -8,12 +8,13 @@ See https://blockprotocol.org/docs/developing-blocks
 
 ## Step two: write and build a component
 
-1.  Change into the folder: `cd packages/hash/blocks/<name>`
+1.  Change into the folder: `cd path/to/your-block-name`
 
-1.  Write a React component starting in `App.tsx`. To test it during development:
+1.  Write a React component starting in `app.tsx`. To test it during development:
 
-    - edit `src/webpack-dev-server.js` to give your component some props to test with
-    - run the dev server with `yarn start`
+    - edit `package.json` → `examples` to give your component some props to test with
```

Comment:
```suggestion
    - edit `package.json` → `blockProtocol` → `examples` to give your component some props to test with. This will also be included as example data in the generated `block-metadata.json`
```

Reply:
👍 317a16d0a8d229984a9793525c06c92fc72037c2 (I wrote `blockprotocol` instead of `blockProtocol`)